src/views/Simulation/components/SceneContainer/SceneContainer.js

```diff
@@ -3,30 +3,21 @@
 import React, { useEffect, useRef } from 'react';
 import PropTypes from 'prop-types';
 import { useTheme } from '@mui/styles';
-import '@pixi/unsafe-eval';
 import flowchartInstance from '../../data/output.json';
-import { createApp, renderElements } from './pixiUtils';
+import { createApp, destroyApp, initApp } from './pixiUtils';
 
 const SceneContainer = ({ toggleInspectorDrawer }) => {
   const theme = useTheme();
 
   const containerRef = useRef(null);
-  const canvasRef = useRef(null);
-  const appRef = useRef(null);
+  const app = createApp();
```

Comment:
Calling `createApp` in the main render function of the component will create a new Pixi `Application` every time the component is re-rendered. Also, these new applications will never be used because `initApp` is called only once when the component is mounted for the first time (in the `useEffect` with the empty dependency array)

Reply:
Initially, I got out the app creation outside the `useEffect` because I thought it caused screen app duplication during each app live update. Later I discovered it was the `useRef` use issue. I forgot to replace the app creation inside the `useEffect`. It's done now.